src/fetchAPI.js

```diff
@@ -15,5 +15,11 @@ export default {
     return fetch(`http://api.airvisual.com/v2/city?city=${city}&state=${state}&country=USA&key=ef754ecf-5ebf-47b6-ae9f-f96d1f8eda21`)
       .then(response => response.json())
       .catch(error => console.log(error))
+  },
+
+  getCurrent () {
+    return fetch('http://api.airvisual.com/v2/nearest_city?key=ef754ecf-5ebf-47b6-ae9f-f96d1f8eda21')
+      .then(response => response.json())
+      .catch(error => console.log(error))
```

Comment:
Love this to get the most current data for our user's actual nearest location! And I don't think we should have any issues with accessing by IP geolocation in the way that we have this set up.